cypress/e2e/onboarding.ts

```diff
@@ -30,7 +30,8 @@ describe('onboarding', () => {
     cy.findByRole('main').within(() => {
       cy.findByRole('textbox', {name: /name/i}).type(firstName)
       cy.findByRole('group', {name: /team/i}).within(() => {
-        cy.findByRole('radio', {name: /blue/i}).click()
+        // checkbox is covered with a <label>
+        cy.findByRole('radio', {name: /blue/i}).click({force: true})
```

Comment:
The radio is wrapped in a label, but that's a good thing. Right?

```
  1) onboarding
       should allow a user to register a new account:
     CypressError: Timed out retrying after 4050ms: `cy.click()` failed because this element:

`<input type="radio" class="sr-only" name="team" value="BLUE">`

is being covered by another element:

`<label class="block pb-12 pt-20 px-12 text-center">...</label>`

Fix this problem, or use {force: true} to disable error checking.
```

Reply:
That's fine 👍 It's accessible and that's all that matters.